.gitignore

```diff
@@ -77,7 +77,6 @@ capybara-*html
 
 #rbenv
 .rbenv-version
-.ruby-version
```

Comment:
I removed to make use of the fact the Travis can automatically use the .ruby-version to determine the Ruby build to use. But it caused issues on 0a because it doesn't have Ruby 2 so I can actually revert this change because I'm not using it anymore.